charters/QueryService/charter.md

```diff
@@ -0,0 +1,71 @@
+
+# Query Service
+
+
+## Description
+The Query Service is an Application Programming Interface (API) for querying Human Cell Atlas (HCA) bundle metadata with different query languages.
+
+## Objectives
+1. Build, operate, and maintain a metadata API query service that allows a variety of queries to be performed against bundle metadata stored in HCA Data Coordination Platform (DCP) Data Storage Service (DSS)
+1. Develop a query language specification and metadata Extract, Transform, Load (ETL) pattern to accommodate intuitive querying
```

Comment:
This seems like two objectives in one, and the ETL pattern is somewhat an implementation detail. The metadata charter (currently proposed) specifically calls out that extending and improving the schema based on feedback (of which intuitive querying would presumably encompass) is in scope. I can see a risk here that query usecases are never propagated back to the metadata team

Reply:
✅:
1) Removing the ETL as a goal assuming it is an implementation detail
2) Adding the metadata group to the feedback section